index.d.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2015-2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+import { OcspModes } from "./lib/core";
+
+/**
+ * The snowflake-sdk module provides an instance to connect to the Snowflake server
+ * @see [source](https://docs.snowflake.com/en/developer-guide/node-js/nodejs-driver)
+ */
+declare module 'snowflake-sdk' {
```

Comment:
I checked in IDEA that the types are not correctly visible e.g. Connection or Statement are not visible and cannot be imported

Maybe we can start with single file with all the types?

Reply:
I checked that imports in index.d.ts seem not to work as expected - having a single file with all types seems to be easier to start

Reply:
@sfc-gh-ext-simba-jy prepared version with single file.